script/deploy/zksync/DeployAcrossFacetV3.zksync.s.sol

```diff
@@ -20,17 +20,21 @@ contract DeployScript is DeployScriptBase {
     }
 
     function getConstructorArgs() internal override returns (bytes memory) {
-        string memory path = string.concat(root, "/config/across.json");
+        string memory configPath = string.concat(root, "/config/across.json");
+        string memory networksPath = string.concat(
+            root,
+            "/config/networks.json"
+        );
 
         address acrossSpokePool = _getConfigContractAddress(
-            path,
+            configPath,
             string.concat(".", network, ".acrossSpokePool")
         );
-        address weth = _getConfigContractAddress(
-            path,
-            string.concat(".", network, ".weth")
```

Comment:
can we remove `weth` field from `across.json` for all chains then?

Reply:
the config is generated by the BE, we should not update it. 
But yes, we can ask the BE to remove it.